src/admin/components/forms/field-types/RadioGroup/RadioInput/index.scss

```diff
@@ -7,7 +7,10 @@
   margin: base(.10) 0;
 
   input[type=radio] {
-    display: none;
+    opacity: 0;
+    width: 0;
+    height: 0;
+    pointer-events: none;
```

Comment:
We should be able to just use `visibility: hidden` here, and then use the neighbor selector to style the styled input.

Reply:
I tested `visibility: hidden` but, similar to `display: none`, the input could not be tabbed through when this is set.

Reply:
Looks good, I just removed some extra margin that was hanging around.